Department-for-Transport-Disruptions/bods-integrated-data
# BODS-4457: Add basic auth to AVL subscribe and unsubscribe endpoints

To test:
- Run `make setup` (to update function and cli-helpers)
- Run `make invoke-avl-subscriber` and run through the script:
   - endpoint:  http://siri.ticketer.org.uk/api/vm
   - username: BODS_AVL_Dela
   - password: message me for password :)

Fingers crossed, you should be subscribed to Delaine Coaches feed! Check the dynamo table to check the subscription is active and to copy the subscription ID to your clipboard. (You won't see data incoming to S3 - think this is because it's running locally - will check data is received as expected when merged into dev)

To unsubscribe, run `make invoke-avl-subscriber` and paste in the subscription ID. Check dynamo to see if status = TERMINATED
